spec/eof.md

```diff
@@ -128,27 +128,21 @@ Code executing within an EOF environment will behave differently than legacy cod
 
 #### Creation transactions
 
-Creation transactions (tranactions with empty `to`), with `data` containing EOF code (starting with `EF00` magic) are interpreted as having a concatenation of EOF `initcontainer` and `calldata` in the `data` and:
+Creation transactions (tranactions with empty `to`), with `data` containing EOF code (starting with `EF00` magic) are interpreted as having a concatenation of an EOF `txcontainer` and `calldata` in the `data` and:
 
-1. intrinsic gas cost rules and limits defined in EIP-3860 for legacy creation transaction apply. The entire `data` of the transaction is used for these calculations
-2. Find the split of `data` into `initcontainer` and `calldata`:
+1. intrinsic gas cost rules and limits defined in EIP-3860 for legacy creation transaction apply. The entire `data` of the transaction is used for these calculations, as if it were the initcode
```

Comment:
Potential Problem 1: applying initcode-specific rules (eip-3860) to something which is not initcode